docs/CHANGELOG.md

```diff
@@ -0,0 +1,332 @@
+## 
+
+**v0.57.0**
+2024-10-22T12:20:21+01:00: add changelog binary that allows to generate a changelog consisting of commit messages between defined tag and HEAD
```

Comment:
Can we please use just `day-month-year` format? I think that we don't need the hours, minutes and seconds personally :)